### PR TITLE
HIVE-26456: Remove stringifyException Method From Storage Handlers

### PR DESCRIPTION
**Pull request by belugabehr**

### CODE REVIEW

**Comment by belugabehr** [comment]
@zabetak Here is another related PR.  This doesn't totally wipe out the usage of stringifyException in these modules, but removes the low-hanging fruit.

File: accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/mr/HiveAccumuloTableInputFormat.java [hunk 1]
```diff
@@ -175,15 +175,8 @@ public InputSplit[] getSplits(JobConf jobConf, int numSplits) throws IOException
...
-      log.error("Could not configure AccumuloInputFormat", e);
-      throw new IOException(StringUtils.stringifyException(e));
-    } catch (AccumuloSecurityException e) {
-      log.error("Could not configure AccumuloInputFormat", e);
-      throw new IOException(StringUtils.stringifyException(e));
-    } catch (SerDeException e) {
-      log.error("Could not configure AccumuloInputFormat", e);
-      throw new IOException(StringUtils.stringifyException(e));
+    } catch (AccumuloException | AccumuloSecurityException | SerDeException e) {
+      throw new IOException("Could not configure AccumuloInputFormat", e);
```
Comment (zabetak) [inline accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/mr/HiveAccumuloTableInputFormat.java:179]: Remove unused `import org.apache.hadoop.util.StringUtils;`

File: accumulo-handler/src/test/org/apache/hadoop/hive/accumulo/predicate/TestAccumuloPredicateHandler.java [hunk 2]
```diff
@@ -386,7 +386,7 @@ public void testPushdownComparisonOptNotSupported() {
     } catch (RuntimeException e) {
       assertTrue(e.getMessage().contains("Unexpected residual predicate: field1 is not null"));
     } catch (Exception e) {
-      fail(StringUtils.stringifyException(e));
+      fail(e.getMessage());
     }
```
[the remark this answers is not in the document]
Comment (zabetak) [reply accumulo-handler/src/test/org/apache/hadoop/hive/accumulo/predicate/TestAccumuloPredicateHandler.java:390]: Remove unused `import org.apache.hadoop.util.StringUtils;`

File: accumulo-handler/src/test/org/apache/hadoop/hive/accumulo/predicate/TestAccumuloPredicateHandler.java [hunk 3]
```diff
@@ -475,7 +474,7 @@ public void testIgnoreIteratorPushdown() throws TooManyAccumuloColumnsException
       List<IteratorSetting> iterators = handler.getIterators(conf, columnMapper);
       assertEquals(iterators.size(), 0);
     } catch (Exception e) {
-      fail(StringUtils.stringifyException(e));
+      fail(e.getMessage());
     }
```
Comment (zabetak) [inline accumulo-handler/src/test/org/apache/hadoop/hive/accumulo/predicate/TestAccumuloPredicateHandler.java:478]: Same comment as before.

File: druid-handler/src/test/org/apache/hadoop/hive/druid/QTestDruidSerDe.java [hunk 4]
```diff
@@ -96,7 +96,7 @@ public class QTestDruidSerDe extends DruidSerDe {
           DruidStorageHandlerUtils.JSON_MAPPER.readValue(RESPONSE, new TypeReference<List<SegmentAnalysis>>() {
           });
     } catch (Exception e) {
-      throw new SerDeException(StringUtils.stringifyException(e));
+      throw new SerDeException(e);
```
Comment (zabetak) [inline druid-handler/src/test/org/apache/hadoop/hive/druid/QTestDruidSerDe.java:99]: Remove unused `import org.apache.hadoop.util.StringUtils;`

File: accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/AccumuloStorageHandler.java [hunk 5]
```diff
@@ -369,11 +369,7 @@ public void preCreateTable(Table table) throws MetaException {
           tableOpts.create(idxTable);
         }
       }
-    } catch (AccumuloSecurityException e) {
-      throw new MetaException(StringUtils.stringifyException(e));
-    } catch (TableExistsException e) {
-      throw new MetaException(StringUtils.stringifyException(e));
-    } catch (AccumuloException e) {
+    } catch (AccumuloSecurityException | TableExistsException | AccumuloException e) {
```
Comment (zabetak) [inline accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/AccumuloStorageHandler.java:372]: In a previous PR (https://github.com/apache/hive/pull/3478) we opted to use the LOG + throw pattern as an alternative of removing `stringifyException` acknowledging advantages and disadvantages of it. Why we don't want to do the same here?

**Pull request closed without merging**